kani-driver/src/call_cargo.rs

```diff
@@ -125,8 +113,8 @@ impl KaniSession {
                     .args(&target.to_args())
                     .args(&pkg_args)
                     .env("RUSTC", &self.kani_compiler)
-                    .env("RUSTFLAGS", "--kani-flags")
-                    .env("KANIFLAGS", &crate::util::join_osstring(&kani_args, " "))
+                    .env("KANIFLAGS", kani_args.join(" "))
+                    .env("CARGO_ENCODED_RUSTFLAGS", rustc_args.join(OsStr::new("\x1f")))
```

Comment:
Can you clarify what this does?

Reply:
It's the same as RUSTFLAGS. Per [their documentation](https://doc.rust-lang.org/cargo/reference/environment-variables.html), it is the recommended way. I don't know if it matters much here, but I figured why not:

> This string is split by whitespace; for a more robust encoding of multiple arguments, see CARGO_ENCODED_RUSTFLAGS.

Reply:
Please add a link to the documentation